### PR TITLE
Adds methods to more easily convert/parse MLS from/to html forms

### PR DESCRIPTION
**Pull request by bwiedmann**

### CODE REVIEW

File: src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java [hunk 1]
```diff
@@ -201,4 +207,19 @@ protected boolean isConsideredNull(Object propertyValue) {
         }
         return values.entrySet().stream().anyMatch(entry -> Strings.isEmpty(entry.getValue()));
     }
+
+    @Override
+    public void parseComplexValues(Object e, Map<String, Value> values) {
```
Comment (andyHa) [inline src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java:212]:
```suggestion
    public void parseComplexValues(Object entity, Map<String, Value> values) {
```

File: src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java [hunk 2]
```diff
@@ -201,4 +207,19 @@ protected boolean isConsideredNull(Object propertyValue) {
...
+        parseValue(e, Value.of(values));
```
Comment (andyHa) [inline src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java:213]: Shouldn't we reather implement the logik here?

File: src/main/java/sirius/db/mixing/types/MultiLanguageString.java [hunk 3]
```diff
@@ -302,4 +303,10 @@ public void setData(Map<String, String> newData) {
     public void remove(String languageKey) {
         super.modify().remove(languageKey);
     }
+
+    public String getAsJSON() {
```
Comment (andyHa) [inline src/main/java/sirius/db/mixing/types/MultiLanguageString.java:307]: javadoc

**Review by andyHa — CHANGES_REQUESTED** [review]

File: src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java (hunk 2 above)
Comment (bwiedmann) [reply src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java]: Parsing is done here how

**Review by andyHa — APPROVED** [review]

File: src/main/java/sirius/db/mixing/Property.java [hunk 4]
```diff
@@ -575,6 +579,26 @@ public void parseValues(Object e, Values values) {
...
+        return Collections.emptyList();
+    }
+
+    /**
+     * Parses the given complex values map and applies it to the given entity if possible.
+     *
+     * @param e      the entity to receive the parsed value
+     * @param values the values to parse and apply
+     */
+    public void parseComplexValues(Object e, Map<String, Value> values) {
```
Comment (sabieber) [inline src/main/java/sirius/db/mixing/Property.java:598]: e -> entity

**Review by sabieber — CHANGES_REQUESTED** [review]

**Review by idlira — APPROVED** [review]

**Review by jmuscireum — APPROVED** [review]

File: src/main/java/sirius/db/mixing/Property.java [hunk 5]
```diff
@@ -575,6 +578,26 @@ public void parseValues(Object e, Values values) {
...
+    }
+
+    /**
+     * Parses the given complex values map and applies it to the given entity if possible.
+     *
+     * @param entity      the entity to receive the parsed value
+     * @param values the values to parse and apply
+     */
+    public void parseComplexValues(Object entity, Map<String, Value> values) {
+        throw new UnsupportedOperationException(getClass().getName() + " must be overriden");
```
Comment (marvinwoop) [inline src/main/java/sirius/db/mixing/Property.java:598]: kleiner Typo: overridden

**Review by sabieber — APPROVED** [review]

**Review by marvinwoop — APPROVED** [review]

**Pull request merged**